tensorrt_llm/_torch/models/modeling_mistral.py

```
@@ -716,6 +716,152 @@ def get_mm_special_token_ids(self) -> torch.Tensor:
+
+    @staticmethod
+    def _vit_tokens(*, width: int, height: int, patch: int) -> int:
+        """ViT attention-sequence length (pre-merge patches) for an image."""
+        return (height // patch) * (width // patch)
+
+    def get_size_for_max_tokens(self, *, max_tokens: int) -> Dict[str, int]:
+        """Largest square Pixtral image (aligned to ``patch * merge``) whose
+        ViT patch count is ``<= max_tokens``.
+
+        Raises ``ValueError`` if even the smallest aligned image
+        (``unit × unit``) exceeds ``max_tokens``.
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)